core/src/main/scala/org/apache/spark/internal/io/SparkHadoopWriterUtils.scala

```diff
@@ -106,6 +110,18 @@ object SparkHadoopWriterUtils {
     }
   }
 
+  def commitJob(
+      job: JobContext,
+      uuid: String,
+      committer: FileCommitProtocol,
+      commitMsgs: Seq[TaskCommitMessage]): Long = {
```

Comment:
The retrun value of `commitJob` seems not be used.

Reply:
> The retrun value of `commitJob` seems not be used.

Will be used in https://github.com/apache/spark/pull/31522, revert this in current pr